spec/components/layout_for_admin_spec.rb

```diff
@@ -16,4 +16,25 @@ def component_name
 
     assert_select 'meta[name="robots"][content="noindex,nofollow,noimageindex"]', visible: false
   end
+
+  it "can receive a custom js filename" do
+    expect_any_instance_of(ActionView::Base).to receive(:javascript_include_tag).with("govuk_publishing_components/vendor/modernizr").once
```

Comment:
Do we need to do some mocking? i.e. does it error without it?

If we do need it, I'd suggest we use an allow here, as this isn't actually an assertion we're making about js_filename, if not I'd just delete it.

(Ideally it won't be too long until modernizr can be removed - I think it's only for IE < 11)

Reply:
Yes unfortunately it blows up without t

Reply:
Sure, can we change it to allow to indicate it's not one of our expectations?

Reply:
oh right sorry i get what you mean 👍 sorry i thought you meant both